internal/scheduler/api.go

```diff
@@ -68,7 +68,7 @@ func (srv *ExecutorApi) StreamingLeaseJobs(stream api.AggregatedQueue_StreamingL
 
 	// Get data stored in sql for these jobs.
 	// In particular, the Pulsar submit job message for each job.
-	jobIds := make([]uuid.UUID, len(runs))
+	jobIds := make([]string, len(runs))
```

Comment:
yes that's a good idea- I'll do that

Reply:
Actually although it's a good idea  the implementation turns out to be horrendous because the rest of the system expects JobIds to be strings, which in turn means we have to either change a gazillion functions to take our new ulid type, or we end up having to cast everywhere in the scheduler.

I think it's best to  kepp it as is for now.  Hopefully once the new scheduler is ourt, we can change the JobIds to be a proper UUID.  That should clear up this mess as everything has UUID support.

Reply:
I agree. Leave it as strings and then move towards UUIDs.